### PR TITLE
Deploy To Azure Using CLI

### PR DESCRIPTION
**Pull request by pderop**
This is a second attempt to deploy the site to Azure (the previous PR #147 was not working correctly).

This time this PR is doing a blue-green deployment (with zero down time) using the az cli directly, and is based on [this example](https://github.com/Azure-Samples/azure-spring-apps-blue-green/blob/main/.github/workflows/blue-green-deploy.yml).

### CODE REVIEW

File: .github/workflows/deploy-to-azure.yml [hunk 1]
```diff
@@ -25,35 +33,47 @@ jobs:
         uses: gradle/gradle-build-action@29c0906b64b8fc82467890bfb7a0a7ef34bda89e # v3
         with:
           arguments: build -x spotlessCheck
-      - name: Login via Azure CLI
-        uses: azure/login@8c334a195cbb38e46038007b304988d888bf676a # v2
+      - name: Login Via Azure CLI
+        uses: azure/login@cb79c773a3cfa27f31f25eb3f677781210c9ce3d # v1
```
Comment (violetagg) [inline .github/workflows/deploy-to-azure.yml:37]: Why are you reverting to version 1?

**Review by violetagg — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy-to-azure.yml (hunk 1 above)
Comment (pderop) [reply .github/workflows/deploy-to-azure.yml:37]: Thanks Violeta for noticing this.
Can we ignore this for the moment and expect Renovate to make subsequent PRs in order to fix all versions pinning ?

Comment (violetagg) [reply .github/workflows/deploy-to-azure.yml:37]: I would like to not ignore and not put noise like this in the PR

Comment (pderop) [reply .github/workflows/deploy-to-azure.yml:37]: ok, then I'll fix this soon.

Comment (pderop) [reply .github/workflows/deploy-to-azure.yml]: applied feedback, see https://github.com/reactor/projectreactor.io/pull/150/commits/e0efd7d55c2b2b2592f27e8701b865196964785a

Comment (pderop) [reply .github/workflows/deploy-to-azure.yml]: there are other dependencies that are not pinned, let me know if you want to fix them too.

Comment (violetagg) [reply .github/workflows/deploy-to-azure.yml]: thanks, now it is obvious that this fallback to version 1 is not intentional!

**Review by violetagg — APPROVED** [review]

File: .github/workflows/deploy-to-azure.yml (hunk 1 above)
Comment (violetagg) [reply .github/workflows/deploy-to-azure.yml]: > there are other dependencies that are not pinned, let me know if you want to fix them too.

not is was just this one, when someone is reviewing the PR it is good to have only relevant changes in the PR and no noise.

**Comment by pderop** [comment]
@violetagg , thanks for your review!

**Pull request merged**